nipype/interfaces/ants/registration.py

```diff
@@ -975,3 +975,89 @@ def _list_outputs(self):
         if len(self.inputs.save_state):
             outputs['save_state'] = os.path.abspath(self.inputs.save_state)
         return outputs
+
+
+
+
+
+class RegistrationSynQuickInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(3, 2, argstr='-d %d',
+                            usedefault=True, desc='image dimension (2 or 3)')
+    fixed_image = InputMultiPath(File(exists=True), mandatory=True, argstr='-f %s',
+                                 desc='Fixed image or source image or reference image')
+    moving_image = InputMultiPath(File(exists=True), mandatory=True, argstr='-m %s',
+                                  desc='Moving image or target image')
+    output_prefix = traits.Str("transform", usedefault=True, argstr='-o %s',
+                               desc="A prefix that is prepended to all output files")
+
+    # todo ANTSCommandInputSpec already has this, but I can't figure out how to set it without defining it again
+    num_threads = traits.Int(default_value=1, desc='Number of threads (default = 1)', argstr='-n %d')
+
+    transform_type = traits.Enum('s', 't', 'r', 'a', 'sr', 'b', 'br', argstr='-t %s',
+                                 desc='transform type\n\
+                                 t:  translation\
+                                 r:  rigid \n\
+                                 a:  rigid + affine\n\
+                                 s:  rigid + affine + deformable syn (default)\
+                                 sr: rigid + deformable syn\
+                                 b:  rigid + affine + deformable b-spline syn\n\
+                                 br: rigid + deformable b-spline syn',
+                                 usedefault=True)
+
+    use_histogram_matching = traits.Bool(default=False, argstr='-j %s',
```

Comment:
`traits.Bool(False, argstr ...)`

additionally: you may remove the `_format_arg` for this input setting `argstr='-j %d'`